src/pages/script/styles/script.css

```
@@ -0,0 +1,561 @@
+.script-page {
+  min-height: 100vh;
+  background: #f8f9fa;
+  font-family: "SUIT", sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SUIT" (font-family-name-quotes)

(font-family-name-quotes)